lib/dotcom/application.ex

```diff
@@ -26,29 +26,16 @@ defmodule Dotcom.Application do
 
     children =
       [
-        # Start the endpoint when the application starts
-        %{
-          id: ConCache,
-          start:
-            {ConCache, :start_link,
-             [
-               [
-                 ttl: :timer.seconds(60),
-                 ttl_check: :timer.seconds(5),
-                 ets_options: [read_concurrency: true]
-               ],
-               [name: :line_diagram_realtime_cache]
-             ]}
-        },
-        RepoCache.Log,
-        {Application.get_env(:dotcom, :cms_cache, CMS.Cache), []},
-        CMS.Telemetry,
-        V3Api.Cache,
-        Schedules.Repo,
-        Schedules.RepoCondensed,
-        Facilities.Repo,
-        Stops.Repo
+        {Application.get_env(:dotcom, :cache, Dotcom.Cache.Multilevel), []},
+        V3Api.Cache
       ] ++
+        if Mix.env() != :test do
```

Comment:
blocking: This is making the application [crash](https://mbta.splunkcloud.com/en-US/app/search/show_source?sid=1709324779.12429448&offset=5&latest_time=1709323093)


```
8eea6c36ed1b ** (exit) {:bad_return, {{Dotcom.Application, :start, [:normal, []]}, {:EXIT, {:undef, [{Mix, :env, [], []}, {Dotcom.Application, :start, 2, [file: ~c"lib/dotcom/application.ex", line: 32]}, {:application_master, :start_it_old, 4, [file: ~c"application_master.erl", line: 293]}]}}}}
8eea6c36ed1b (kernel 9.2) application_master.erl:142: :application_master.init/4
8eea6c36ed1b (stdlib 5.2) proc_lib.erl:241: :proc_lib.init_p_do_apply/3
8eea6c36ed1b Initial Call: :application_master.init/4
8eea6c36ed1b Ancestors: [#PID<0.2846.0>]
8eea6c36ed1b [0m[22m2024-03-01T19:58:11.782 [notice] node=dotcom@ip-10-2-16-141 Application dotcom exited: exited in: Dotcom.Application.start(:normal, [])
8eea6c36ed1b ** (EXIT) an exception was raised:
8eea6c36ed1b ** (UndefinedFunctionError) function Mix.env/0 is undefined (module Mix is not available)
8eea6c36ed1b Mix.env()
8eea6c36ed1b (dotcom 0.0.1) lib/dotcom/application.ex:32: Dotcom.Application.start/2
8eea6c36ed1b (kernel 9.2) application_master.erl:293: :application_master.start_it_old/4
8eea6c36ed1b Kernel pid terminated (application_controller) ("{application_start_failure,dotcom,{bad_return,{{'Elixir.Dotcom.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.Mix',env,[],[]},{'Elixir.Dotcom.Application',start,2,[{file,\"lib/dotcom/application.ex\"},{line,32}]},{application_master,start_it_old,4,[{file,\"application_master.erl\"},{line,293}]}]}}}}}")
8eea6c36ed1b Crash dump is being written to: erl_crash.dump...done

```